thunderfish/psdtype.py

```diff
@@ -0,0 +1,173 @@
+import numpy as np
+import sys
+
+def bin_it(power, freq_bins, max_freq, res):
+    """
```

Comment:
this 3-liner could go directly into the function where you use it, with an one line comment explaining what it does